.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.10", "3.11","3.12", "3.13"]
+        python-version: ["3.11","3.12", "3.13"]
```

Comment:
This is fine, 3.10 is already dropped by SPEC-0: https://scientific-python.org/specs/spec-0000/